src/RoboFile/ClientContainer.php

```diff
@@ -42,7 +42,24 @@ public function runContainerTests(
 		$opts['install-test'] = '';
 
 		$this->runContainerTestPreparation($opts);
+
 		$this->runTestSuites($opts);
+
+	}
+
+	public function runContainerTest(
```

Comment:
why don't we simply rename the function or create those two variants as you mentioned?